src/NHSD.BuyingCatalogue.Identity.UserDatabase/Tables/Organisations.sql

```diff
@@ -1,7 +1,8 @@
 ﻿CREATE TABLE [dbo].[Organisations]
 (
-	[Id] UNIQUEIDENTIFIER NOT NULL PRIMARY KEY, 
-    [Name] VARCHAR(255) NOT NULL, 
-    [OdsCode] VARCHAR(8) NULL, 
-    [LastUpdated] DATETIME2 NOT NULL
-)
+	Id UNIQUEIDENTIFIER NOT NULL,
+    [Name] NVARCHAR(255) NOT NULL,
+    OdsCode NVARCHAR(8) NULL,
+    LastUpdated DATETIME2 NOT NULL DEFAULT GETUTCDATE(),
```

Comment:
Not sure about the default GETUTCDATE() as I think by leaving this out ensures the application code has to specify a datetime & would give more flexibility.

Reply:
As `LastUpdated` will always be the current date on creation this suggestion was just to avoid the need to provide the current date on creation. Removing the default wouldn't stop `LastUpdated` from being omitted in a subsequent update but it's not going to cause any problems either so can be removed if preferred.